e2e-tests/gatsby-image/cypress/integration/fixed.js

```diff
@@ -23,12 +23,12 @@ describe(`fixed`, () => {
   })
 
   // TODO: figure out why these tests are failing
```

Comment:
```suggestion
```

Reply:
nit (so won't block the PR!) but we should probably remove this, since they are passing 😅